README.md

```diff
@@ -21,7 +21,7 @@ torchao just works with `torch.compile()` and `FSDP2` over most PyTorch models o
 
 ### Post Training Quantization
 
-Quantizing and Sparsifying your models is a 1 liner that should work on any model with an `nn.Linear` including your favorite HuggingFace model. You can find a more comprehensive usage instructions [here](torchao/quantization/), sparsity [here](/torchao/_models/sam/README.md) and a HuggingFace inference example [here](scripts/hf_eval.py)
+Quantizing and Sparsifying your models is a 1 liner that should work on any model with an `nn.Linear` including your favorite HuggingFace model. You can find a more comprehensive usage instructions [here](torchao/quantization/), sparsity [here](/torchao/_models/sam/README.md) and a HuggingFace inference example [here](https://huggingface.co/docs/transformers/main/en/quantization/torchao?torchao=manual#torchao).
```

Comment:
we can clarify this a bit by specifying user can either use torchao native APIs or quantizing huggingface model directly with configs here

Reply:
Sure, added some details on the 2 methods with examples. Let me know what you think.

Reply:
can you add the example for point 1 and 2 like:
1. Using torchao APIs directly. ....(torchao/quantization/), sparsity [here](/torchao/_models/sam/README.md)
2. Loading a huggingface model with a quantization config. ...(https://huggingface.co/docs/transformers/main/en/quantization/torchao?torchao=manual#torchao).